### PR TITLE
NO-JIRA: faq: add more details and examples for kernel <--> kernel-rt

### PR DESCRIPTION
**Pull request by jlebon**
This comes up often enough that it's worth elaborating.

### CODE REVIEW

**Review by yasminvalim — APPROVED** [review]
LGTM

**Review by aaradhak — APPROVED** [review]
/lgtm

File: docs/faq.md [hunk 1]
```diff
@@ -386,14 +386,57 @@ If the package is **not** included in Fedora CoreOS, you may submit a PR to this
...
-rpm-ostree override remove kernel kernel-core kernel-modules kernel-modules-extra --install kernel-rt-core-4.18.0-305.34.2.rt7.107.el8_4.x86_64.rpm --install kernel-rt-kvm-4.18.0-305.34.2.rt7.107.el8_4.x86_64.rpm --install kernel-rt-modules-4.18.0-305.34.2.rt7.107.el8_4.x86_64.rpm --install kernel-rt-modules-extra-4.18.0-305.34.2.rt7.107.el8_4.x86_64.rpm
+
+Otherwise, to exactly undo just the kernel -> kernel-rt transition:
+
+```
+rpm-ostree override reset kernel kernel-core kernel-modules kernel-modules-extra \
+  --uninstall kernel-rt-core \
+  --uninstall kernel-rt-kvm \
+  --uninstall kernel-rt-modules \
+  --uninstall kernel-rt-modules
```
Comment (jbtrystram) [inline docs/faq.md:424]: This should be ´--uninstall kernel-rt-modules-extra´

**Review by jbtrystram — COMMENTED** [review]
Maybe also add a note that it's the same process for ´kernel-64k´ ?

File: docs/faq.md (hunk 1 above)
Comment (jlebon) [reply docs/faq.md:424]: Yes, good catch. Fixed in https://github.com/openshift/os/pull/1470.

**Pull request merged**